CONTRIBUTING.md

```diff
@@ -2,18 +2,42 @@
 
 Operator Lifecycle Manager is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into operator-lifecycle-manager.
 
-# Email and Chat
+## Communication
 
-- Email: [operator-framework][operator_framework]  
+- Slack: [#olm-dev on the Kubernetes Slack](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [OLM Dev Google Group](https://groups.google.com/u/1/g/operator-framework-olm-dev)
+- Mailing List: [operator-framework-olm-dev@googlegroups.com](operator-framework-olm-dev@googlegroups.com)
+- Scheduled upstream meetings can be found at the [operator-framework/community GitHub Repository][community]
+
+> Note: By joining the Google Group you will automatically be added to the mailing list.
 
 ## Getting started
 
-- Fork the repository on GitHub
+- Fork and clone the repository on GitHub
+- Review the design documentation in the [doc/design](./doc/design) directory
+- Watch the content on the [Operator Framework Youtube Channel][of-youtube]
 - See the [developer guide](./DEVELOPMENT.md) for build instructions
+- Join the existing [Communication][communication] channels and introduce yourself to the other OLM Developers
 
 ## Reporting bugs and creating issues
 
-Reporting bugs is one of the best ways to contribute. However, a good bug report has some very specific qualities, so please read over our short document on [reporting bugs](./doc/dev/reporting_bugs.md) before submitting a bug report. This document might contain links to known issues, another good reason to take a look there before reporting a bug.
+If any part of the operator-lifecycle-manager project has bugs or documentation mistakes, please let us know by [opening an issue][operator-olm-issue]. We treat bugs very seriously and believe no issue is too small. Before creating a bug report, please check that an issue reporting the same problem does not already exist. If you aren't sure if the observed behavior is a bug, please reach out to the OLM team using the links provided under [Communication][communication].
+
+To make the bug report accurate and easy to understand, please try to create bug reports that are:
+
+- Specific. Include as much details as possible: which version, what environment, what configuration, etc.
```

Comment:
```suggestion
- Specific. Include as much detail as possible: which version, what environment, what configuration, etc.
```

Reply:
The suggested change has been accepted in the latest version of the pull request.